docs/guides/sre/configuring-ssl.md

```diff
@@ -0,0 +1,440 @@
+            <port>9444</port>
+        </server>
+        <server>
+            <id>2</id>
+            <hostname>chnode2.marsnet.local</hostname>
+            <port>9444</port>
+        </server>
+        <server>
+            <id>3</id>
+            <hostname>chnode3.marsnet.local</hostname>
+            <port>9444</port>
+        </server>
+    </raft_configuration>
+</keeper_server>
+
+```
+
+2. Uncomment and update zookeeper settings on all nodes and set `<secure>` flag
+```xml
+    <zookeeper>
+        <node>
+            <host>chnode1.marsnet.local</host>
+            <port>9281</port>
+            <secure>1</secure>
+        </node>
+        <node>
+            <host>chnode2.marsnet.local</host>
+            <port>9281</port>
+            <secure>1</secure>
+        </node>
+        <node>
+            <host>chnode3.marsnet.local</host>
+            <port>9281</port>
+            <secure>1</secure>
+        </node>
+    </zookeeper>
+```
+
+3. Update and add the following cluster settings to `chnode1` and `chnode2`. `chnode3` will be used for ClickHouse Keeper quorum.
+:::note
+For this configuration example, only one example cluster is configured. The test sample clusters must be either removed, commented out or, if there is an existing cluster exists that is being tested, the port must be updated and the `<secure>` option must be added. The `<user` and `<password>` must be set if the `default` user was initially configured to have a password in the installation or in the `users.xml` file.
+:::
+
+
+The following creates a cluster with one shard replica on two servers (one on each node).
+```xml
+    <remote_servers>
+        <cluster_1S_2R>
+            <shard>
+                <replica>
+                    <host>chnode1.marsnet.local</host>
+                    <port>9440</port>
+                    <user>default</user>
+                    <password>ClickHouse123!</password>
+                    <secure>1</secure>
+                </replica>
+                <replica>
+                    <host>chnode2.marsnet.local</host>
+                    <port>9440</port>
+                    <user>default</user>
+                    <password>ClickHouse123!</password>
+                    <secure>1</secure>
+                </replica>
+            </shard>
+        </cluster_1S_2R>
+    </remote_servers>
+```
+
+4. Set macros values to be able to create a replicatedMergeTree table to test
+
+On `chnode1`
+```xml
+    <macros>
+        <shard>1</shard>
+        <replica>replica_1</replica>
+    </macros>
+```
+
+On `chnode2`
+```xml
+    <macros>
+        <shard>1</shard>
+        <replica>replica_2</replica>
+    </macros>
+```
+
+
+## 5. Configure SSL-TLS interfaces on ClickHouse nodes 
+Settings below configured in the clickhouse-server `config.xml`
+
+1.  Set the display name for the deployment (optional)
+```xml
+<display_name>clickhouse</display_name>
+```
+
+2. Set ClickHouse to listen on external ports
```

Comment:
Maybe switch listen host to `<listen_host>::</listen_host>` to listen on both IPv4 and V6

Reply:
good idea. will this setting do both for interfaces? just didn't  wan't it just listen on localhost or add extra steps for configuring everything on IPv6, I think we can make that into a separate `advanced` doc instead of a `getting started` type one.